src/renderer/entities/proxy/lib/utils.ts

```diff
@@ -124,9 +122,7 @@ const ProxyTypeName: Record<ProxyType, string> = {
   [ProxyType.NOMINATION_POOLS]: 'proxy.names.nominationPools',
 };
 
-function getProxyTypeName(proxyType: ProxyType | string, t: TFunction) {
+function getProxyTypeName(proxyType: ProxyType | string): string {
   // if proxy type is not in ProxyTypeName enum split camel case string and add spaces
```

Comment:
This comment redundant because we have a properly called `splitCamelCaseString `